spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-actuator/src/main/resources/application.properties

```diff
@@ -31,3 +31,11 @@ management.endpoints.migrate-legacy-ids=true
 
 management.endpoints.jackson.isolated-object-mapper=true
 spring.jackson.visibility.field=any
+
+#management.tracing.sampling.probability=1.0
```

Comment:
I left these changes in samples intentionally in the PR (it's a separate commit), one the PR is ready to merge I will drop it.